new-log-viewer/src/utils/data.ts

```diff
@@ -38,7 +38,55 @@ const getMapValueWithNearestLessThanOrEqualKey = <T>(
         map.get(lowerBoundKey) as T;
 };
 
+/**
+ * Creates an array of numbers in the range `[begin, end)` with the `i`th element computed as
+ * `range[i - 1] + step`.
+ *
+ * If only one argument is supplied, the argument is interpreted as `end`, and `begin` is set to
+ * `0`.
```

Comment:
```suggestion
 * If `args` is a number, it is interpreted as `end`, and `begin` is set to `0`.
```

Feel free to disagree.